python/mxnet/ndarray/sparse.py

```diff
@@ -786,6 +804,14 @@ def csr_matrix(arg1, shape=None, ctx=None, dtype=None):
             - **dtype** (*str or numpy.dtype, optional*) - The data type of the output array. \
             The default dtype is float32.
 
+    - csr_matrix((data, (row, col)))
+        to construct a CSRNDArray based on the COOrdinate format \
+        using three seperate arrays, \
+        where ``row[i]`` is the row index of the element, \
+        ``col[i]`` is the column index of the element \
+        and ``data[i]`` is the data corresponding to the element. All the missing \
+        elements in the input are taken to be zeroes.
+
     Parameters
     ----------
     arg1: tuple of int, tuple of array_like, array_like, CSRNDArray or scipy.sparse.csr_matrix
```

Comment:
We should also add scipy.sparse.coo_matrix to type of arg1

Reply:
Added